packages/cardpay-sdk/sdk/constants.ts

```diff
@@ -89,7 +89,8 @@ const constants: {
   sokol: SOKOL,
   kovan: KOVAN,
   mainnet: MAINNET,
-  xdai: XDAI,
+  gnosis: GNOSIS,
+  xdai: GNOSIS,
 });
 
 export const networks: { [networkId: number]: string } = Object.freeze({
```

Comment:
a few lines down from here is one other place that needs updating; if you visit [the `web-client` productionpreview deployment] there‘s this error in the console:

```
Error: chainId is required to set custom session storage for parallel connections in WalletConnect
```

Reply:
Actually, I think ssr-web is good but web-client still has issues.

Reply:
@backspace Now this should be all good.